framework/components/ecl-accordions/ecl-accordions.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ecl-accordions" role="tablist" aria-multiselectable="true">
+  <h2 id="accordion-header-1" class="ecl-accordions__header" role="tab" aria-controls="accordion-panel-1" tabindex="0" aria-selected="true" aria-expanded="true">
+    <span class="ecl-accordions__header-icon ecl-icon ecl-icon--rounded ecl-u-bg-secondary ecl-icon--growth"></span> Jobs, Growth, Investment and Competitiveness with an additional quite long string
+  </h2>
+  <div id="accordion-panel-1" class="ecl-accordions__panel" role="tabpanel" aria-labelledby="accordion-header-1" tabindex="0" aria-hidden="false">
```

Comment:
What is the reason to set a tabindex here? It somehow mess with keyboard navigation (when using tabs)